src/helper/frontent-origin.ts

```diff
@@ -0,0 +1,9 @@
+export const frontendOrigin =
+  process.env.NEXT_PUBLIC_ENV === 'production'
+    ? // we are on production branch
```

Comment:
production includes production and staging, right?
maybe the comment could say that.

Reply:
(I'd do it, but i'm not sure I'm right :)

Reply:
Actually, in this case production is only production (e.g. de.serlo.org), maybe you have an idea how to make this clearer?

Reply:
production branch -> production environment?

Reply:
Is staging handled in the second case then?

Reply:
(in other words I don't understand how staging is handled here atm. :)

Reply:
staging is technically a preview deployment (we have no custom domain here)

Reply:
Thanks for clarifying!